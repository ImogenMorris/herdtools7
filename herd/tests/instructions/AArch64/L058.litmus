AArch64 L058
{
  0:X0=P1:L0;
  0:X2=instr:"B .+4";
}
 P0          | P1     ;
 LDR W1,[X0] |  B end ;
 CMP W2,W1   | L0:    ;
 B.NE Lend   |  B L1  ;
 MOV W3,#1   | L1:    ;
Lend:        | end:   ;
locations [0:X1;0:X2]
exists (0:X3=0)