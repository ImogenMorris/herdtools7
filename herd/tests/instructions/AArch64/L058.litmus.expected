Test L058 Allowed
States 1
0:X1=instr:"B .+4"; 0:X2=instr:"B .+4"; 0:X3=1;
No
Witnesses
Positive: 0 Negative: 1
Condition exists (0:X3=0)
Observation L058 Never 0 1
Hash=fc2741988c504fee73989498f21e3de5

